.github/workflows/buildcheck.yml

```diff
@@ -0,0 +1,14 @@
+name: Check Build on PR
```

Comment:
With the forked repositories, we could have name prefixed with `tii-`, for example `tii-pr.yml`. That's easier for the maintainer.